fv3core/stencils/neg_adj3.py

```diff
@@ -206,99 +180,131 @@ def fix_neg_cloud(dp: sd, qcld: sd):
 
 
 # Nonstencil code for reference:
-def fix_water_vapor_nonstencil(grid, qv, dp):
+def fix_water_vapor_nonstencil(grid, qvapor, dp):
```

Comment:
Should we comment the non-stencil code?

Reply:
Oops! I'd also be ok with just removing it at this point, though

Reply:
Sounds good, we can save that for the next round too :)